Ginger/GingerCoreNET/Run/RunListenerLib/CenteralizedExecutionLogger/AccountReportApiHandler.cs

```diff
@@ -362,5 +368,65 @@ private async Task<bool> SendRestRequestAndGetResponse(string api, AccountReport
                 return false;
             }
         }
+
+        public List<AccountReportBusinessFlow> GetBusinessflowExecutionDataToCentralDB(Guid executionId)
+        {
+            List<AccountReportBusinessFlow> accountReportBusinessFlows = new List<AccountReportBusinessFlow>();
+            if (restClient != null)
+            {
+                RestRequest restRequest = (RestRequest)new RestRequest(Get_BUSINESSFLOW_EXECUTION_DATA + executionId, Method.Get);
+                string message = string.Format("execution id : {0}", executionId);
+                try
+                {
+                    RestResponse response = restClient.Execute(restRequest);
+                    if (response.IsSuccessful)
+                    {
+                        Reporter.ToLog(eLogLevel.DEBUG, "Successfully validated execution id " + message);
+                        accountReportBusinessFlows = JsonConvert.DeserializeObject<List<AccountReportBusinessFlow>>(response.Content);
+                        return accountReportBusinessFlows;
+                    }
+                    else
+                    {
+                        Reporter.ToLog(eLogLevel.ERROR, "Failed to validate " + message + "Response: " + response.Content);
+                    }
+                }
+                catch (Exception ex)
+                {
+                    Reporter.ToLog(eLogLevel.ERROR, "Exception while validating execution id " + message, ex);
+                }
+                return accountReportBusinessFlows;
+            }
+            return accountReportBusinessFlows;
+        }
+
+        public List<RunsetHLInfoResponse> GetRunsetExecutionDataToCentralDB(Guid executionId)
```

Comment:
If we are getting data from Central DB, please change function name to have 'From' instead of 'To'.